sklearn/metrics/_ranking.py

```diff
@@ -1408,6 +1409,14 @@ def _tie_averaged_dcg(y_true, y_score, discount_cumsum):
     return (ranked * discount_sums).sum()
 
 
+def _check_length_greater_1(y_true: numpy.ndarray):
```

Comment:
Since we are only calling the function once, we can directly put the body of this function where it is called.

By the way, we are old fashion in scikit-learn and we don't use typing yet.

Reply:
Inlined.

My 2 cents: type hints would greatly enhance usability, both as a user of sklearn and as a contributor!

Reply:
We have an issue open about it: https://github.com/scikit-learn/scikit-learn/issues/16705
As far as I remember and in my opinion, we are still missing some integration with the documentation generation to make it really useful for the entire project. But, we are against adding this support in the future.